python/ray/autoscaler/commands.py

```diff
@@ -157,6 +159,13 @@ def kill_node(config_file, yes, override_cluster_name):
     return node_ip
 
 
+def monitor_cluster(cluster_config_file, override_cluster_name):
+    """Kills a random Raylet worker."""
+    cmd = "tail -n 100 -f /tmp/ray/session_*/logs/monitor*"
```

Comment:
Could make the tail length an argument. Sometimes 100 lines doesn't really give you much information and you want to see logs further back.

Reply:
Done (#5082)